p2p_wallet/Scenes/Main/Send/Models/SendSource.swift

```diff
@@ -0,0 +1,4 @@
+enum SendSource: String {
+    case sell
```

Comment:
I don't really think that it is good for Send to know SendSource, Send shouldn't be aware of Sell or anything.

Reply:
I agree but the only purpose of it is analytics. Please look at SendInputViewModel extension. It sends source value as param for all events

Reply:
Ok